vipin/001_draw_grid.py

```diff
@@ -0,0 +1,85 @@
+def draw_grid(n, m):
+    tmpstr = ""
+    if m >= 1:
+        for j in range(0, n+1):
+            for i in range(0, m+1):
+                if i is not m:
+                    tmpstr += "+ - "
+                else:
+                    tmpstr += "+" + "\n"
+                if i == m:
```

Comment:
doesn't need `if i == m` check. the block can be placed in the previous else block